cypress.config.js

```diff
@@ -45,4 +46,10 @@ module.exports = defineConfig({
     // Default is 0
     openMode: 0,
   },
+
```

Comment:
we're only running component tests in this library at the moment, so lets revert changes to this file.

Reply:
Just resolved by reverting this file!